controllers/apps/cluster_status_event_handler.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/apecloud/kubeblocks/controllers/k8score"
 	"github.com/apecloud/kubeblocks/internal/constant"
 	intctrlutil "github.com/apecloud/kubeblocks/internal/controllerutil"
-	probeutil "github.com/apecloud/kubeblocks/internal/sqlchannel/util"
+	lorryutil "github.com/apecloud/kubeblocks/lorry/util"
```

Comment:
@leon-inf @nayutah @xuriwuyun do make sure you guys are following Golang's project layout:

https://github.com/golang-standards/project-layout